src/nrnoc/secref.cpp

```diff
@@ -73,6 +73,15 @@ static double s_unname(void* v) {
 #endif
     pitm = sec2pitm(sec);
     *pitm = (hoc_Item*) 0;
+    auto* ob = sec->prop->dparam[6].get<Object*>();
+    if (ob) {
+        // Avoid use of freed memory in CellBuild when making a section
+        // after deleting a section when continuous create is on.
+        // CellBuild repeatedly creates a single CellBuildTopology[0].dummy
+        // section and then unnames and renamed it at the top level.
+        ob->secelm_ = nullptr;
+        sec->prop->dparam[6] = static_cast<Object*>(nullptr);
```

Comment:
(just a nitpick: would have been nice to have name to the fields than number like `6`)

Reply:
I agree but I think should be done in a separate PR with an enum for all of the
```
section[i].prop->dparam[0].sym  pointer to section symbol
            [1].val     position (0--1) of connection to parent
            [2].val     length of section in microns
            [3].val     first node at position 0 or 1
            [4].val rall branch
            [5].i       aray index
            [6].obj pointer to the object pointer  
            [7].val Ra
            [8].itm hoc_Item* with Section* as element
            [9]._pvoid NrnThread*
            [PROP_PY_INDEX].pvoid pointer to the Python Section object